lib/policyEvaluator/utils/actionMaps.ts

```diff
@@ -233,6 +233,9 @@ const actionMapScuba = {
     AdminStopIngest: 'scuba:AdminStopIngest',
     AdminReadRaftCseq: 'scuba:AdminReadRaftCseq',
     AdminTriggerRepair: 'scuba:AdminTriggerRepair',
+    AdminStartDownsample: 'scuba:AdminStartDownsample',
+    AdminStopDownsample: 'scuba:AdminStopDownsample',
```

Comment:
I think pause/resume is more representative than start/stop, but let's keep it consistent with {start|stop}ingest for now and we can decide at some point before release if we want to change both.